extras/tools/delete_unowned_bricks.sh

```diff
@@ -0,0 +1,266 @@
+#!/bin/bash
+
+set -o pipefail
+
+dryRun="no"
+lvlist=""
+prompt="no"
+vg_id=""
+ip_prefix=""
+fstab_path=""
+
+
+usage() {
+    echo "Usage: $0 [...]"
+    echo ''
+    echo 'The options that this script accepts are:'
+    echo ''
+    echo '--dryrun          shows the commands that would be executed'
+    echo '--lvlist          has the list of volumes to delete'
+    echo '--prompt          prompts for user input before executing every command'
+    echo '--vgid            vgid containing lvs'
+    echo '--ip-prefix       IP address prefix'
+    echo '--fstab-path      provide the path for the fstab file'
+    echo '-h, --help        show this help text'
+    echo ''
+}
+
+
+function parse_args () {
+    args=$(getopt \
+              --options h \
+              --long dryrun,lvlist:,vgid:,prompt,ip-prefix:,fstab-path:,help \
+              -n 'delete_unowned_bricks.sh' \
```

Comment:
I am not sure if we should have basename $0 here or explicitly name it.

Reply:
also $0, afaiac, I just didn't spot this in my first review.

Reply:
use `$0`